main.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+#include <iomanip>
+
+// Stock Program
+// By Clayton Checkeye for CPT_180
+
+int main() {
+        // Declarations
+
+    const int SHARES_BOUGHT = 1000;
+    const int SHARES_SOLD = 1000;
+    const double COMMISSION_RATE = 0.2;
+    const double PURCHASE_PRICE = 45.50;
+    const double SALE_PRICE = 56.90;
+
```

Comment:
Good job declaring your constants toward the top of your program! I forgot to do that.